.changes/next-release/bugfix-c8a42240-0770-442c-b342-f43e3ee0c768.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
```

Comment:
Just in case, does #764 need another changelog entry?

Reply:
Nah, its implementation detail I think